tests/unit_tests/c_wrappers/res/enkf/test_enkf_main.py

```diff
@@ -234,7 +229,7 @@ def test_that_having_no_refcase_but_history_observations_causes_exception(tmpdir
         with open("time_map.txt", "w", encoding="utf-8") as fo:
             fo.writelines("2023-02-01")
 
-        res_config = ResConfig("config.ert")
```

Comment:
This file keeps using `ErtConfig.from_file("config.ert")` where `config.ert` contains:
```
        JOBNAME my_name%d
        NUM_REALIZATIONS 10
        OBS_CONFIG observations
        TIME_MAP time_map.txt
```
we can substitute it with
```python
@pytest.fixture
def observation_config():
    return ErtConfig(model_config=ModelConfig(obs_config_file="observations", time_map_file="time_map.txt"))
```
and separate conserns. It isolates the tests though, at the expense of less coverage here, but I think we have other tests that cover the remaining functionality well.

Reply:
I think this can be done in a separate PR after this is in.